### ISSUE TITLE
Format LV Error as JSON Data - add message

### ISSUE THREAD

**Issue by pollockm** [NISystemsEngineering/JSONRPC#11]
Currently only get the code and source.  Makes for easier system debugging (assuming error files are on the target) if the message is inline with the error report.

Like this:
![image](https://user-images.githubusercontent.com/5288961/40021897-dad46b18-578b-11e8-9028-5f5509aa93d3.png)

**Comment by tannerblair** [comment]
This was omitted for performance reasons. The native error handlers are non-reentrant and have performance issues on RT, so I would like to avoid using them.

**Comment by pollockm** [comment]
Fair enough.  Can it be a toggle option then as to whether it is used?  Not having to look up every error code is a godsend when debugging a flaky system.

**Comment by tannerblair** [comment]
I'm open to it. I agree with the pain point, let me think on the best way to implement it.
